### PR TITLE
bwhite requesting GPU node for image analysis

### PR DESCRIPTION
**Pull request by bswhite**

### CODE REVIEW

File: config/prod/bwhite-iawg.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v9.j2
```
Comment (thomasyu888) [inline config/prod/bwhite-iawg.yaml]: Use new template path

**Comment by thomasyu888** [comment]
Copy and paste entire new template, because there are more than just that line of change

File: config/prod/bwhite-iawg.yaml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Provision an EC2 instance
 template_path: remote/managed-ec2-linux-v1.j2
 stack_name: bwhite-iawg
+sceptre_user_data:
+  Distro: "aws"     # (valid values: aws, ubuntu) AMIId must match distro
```
Comment (thomasyu888) [inline config/prod/bwhite-iawg.yaml]: Did you want to use amazon linux 2 or ubuntu?

Comment (bswhite) [reply config/prod/bwhite-iawg.yaml]: Thanks--this is for a different project. But, yes, let me switch to ubuntu since I'm more familiar with that.

Comment (thomasyu888) [reply config/prod/bwhite-iawg.yaml]: Ya just do `Distro: "ubuntu"` And make sure you update the AMIId below

File: config/prod/bwhite-iawg.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "CSBC-image-analysis"
+  # The resource owner
+  OwnerEmail: "brian.white@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  AMIId: "ami-08333c98cd3b5ada7"
```
Comment (thomasyu888) [inline config/prod/bwhite-iawg.yaml]: Is this an ubuntu distro you built?

**Comment by bswhite** [comment]
It's a ubuntu image that is distributed on the bioconductor site. It is the one I generally use. Khai ran the aws command below (I can't seem to), which gives more info:


>> aws ec2 --output table describe-images --image-ids ami-08333c98cd3b5ada7  

------------------------------------------------------------------------------
|                               DescribeImages                               |
+----------------------------------------------------------------------------+
||                                  Images                                  ||
|+---------------------+----------------------------------------------------+|
||  Architecture       |  x86_64                                            ||
||  CreationDate       |  2019-05-13T16:10:44.000Z                          ||
||  Description        |  bioc-3.9-ubuntu-18.04-40GB-20190513               ||
||  EnaSupport         |  True                                              ||
||  Hypervisor         |  xen                                               ||
||  ImageId            |  ami-08333c98cd3b5ada7                             ||
||  ImageLocation      |  555219204010/bioc-3.9-ubuntu-18.04-40GB-20190513  ||
||  ImageType          |  machine                                           ||
||  Name               |  bioc-3.9-ubuntu-18.04-40GB-20190513               ||
||  OwnerId            |  555219204010                                      ||
||  Public             |  True                                              ||
||  RootDeviceName     |  /dev/sda1                                         ||
||  RootDeviceType     |  ebs                                               ||
||  SriovNetSupport    |  simple                                            ||
||  State              |  available                                         ||
||  VirtualizationType |  hvm                                               ||
|+---------------------+----------------------------------------------------+|
|||                           BlockDeviceMappings                          |||
||+------------------------------------+-----------------------------------+||
|||  DeviceName                        |  /dev/sda1                        |||
||+------------------------------------+-----------------------------------+||
||||                                  Ebs                                 ||||
|||+--------------------------------+-------------------------------------+|||
||||  DeleteOnTermination           |  True                               ||||
||||  Encrypted                     |  False                              ||||
||||  SnapshotId                    |  snap-01cebeb95a682de1c             ||||
||||  VolumeSize                    |  40                                 ||||
||||  VolumeType                    |  gp2                                ||||
|||+--------------------------------+-------------------------------------+|||

**Pull request merged**